saleor/payment/gateways/np_atobarai/errors.py

```diff
@@ -0,0 +1,233 @@
+from typing import Dict, Iterable, List
+
+UNKNOWN_ERROR = "Unknown error while processing the payment."
+
+
+TRANSACTION_REGISTRATION_RESULT_ERRORS = {
+    "E0100048": "Please make sure that the your (Kanji) has been entered.",
+    "E0100049": (
+        "Please confirm that your name(Kanji) does not include prohibited characters."
+    ),
+    "E0100050": (
+        "Please confirm that your name(Kanji) is within 21 full - width characters."
+    ),
+    "E0100051": (
+        "Please confirm that your name(Kana) is within 25 full - width characters."
+    ),
+    "E0100052": (
+        "Please confirm that your name(kana) does not contain prohibited characters."
+    ),
+    "E0100053": (
+        "Please confirm that your company name is within 30 full - width characters."
+    ),
+    "E0100054": (
+        "Please confirm that your company name does not include prohibited characters."
+    ),
+    "E0100055": (
+        "Please confirm that your department name is within 30 full - width characters."
+    ),
+    "E0100056": (
+        "Please confirm that your department name "
+        "does not include prohibited characters."
+    ),
+    "E0100057": "Please check if the customer’s ZIP code has been entered.",
+    "E0100058": (
+        "Please confirm that the customer’s ZIP code "
+        "does not contain prohibited characters."
+    ),
+    "E0100136": (
+        "Please check if the customer’s ZIP code "
+        "format is correct.(Example: 1070052, 107 - 0052)"
+    ),
+    "E0100059": "Please check if the customer’s ZIP code and address match.",
+    "E0100060": "Please make sure that the customer’s address has been entered.",
+    "E0100061": (
+        "Please make sure that the customer’s address "
+        "does not contain prohibited characters."
+    ),
+    "E0100062": (
+        "Please make sure that the Customer’s address "
+        "is within 55 full - width characters."
+    ),
+    "E0100063": "Please check if the customer’s phone number has been entered.",
+    "E0100064": (
+        "Please confirm that the customer’s phone number "
+        "does not contain prohibited characters."
+    ),
+    "E0100065": "Please confirm that the formant of the phone number is valid.",
```

Comment:
```suggestion
    "E0100065": "Please confirm that the format of the phone number is valid.",
```

Reply:
I would recommend checking these error messages with Grammarly or some other tool like this to fix grammar issues.